src/navigation/NavigationTypes.ts

```diff
@@ -16,14 +16,11 @@ interface ExerciseParams {
   closeExerciseAction: CommonNavigationAction
 }
 
-export type ExercisesParams =
-  | {
-      documents: null
-      discipline: Discipline
-      disciplineTitle: string
-      disciplineId: number
-    }
-  | ExerciseParams
+export interface ExercisesParams extends Omit<ExerciseParams, 'documents' | 'closeExerciseAction'> {
```

Comment:
@SteffiStuffel , @klinzo 
i'm not sure if this extend and omit is a good way or its better to create a separate `ExercisesParams` interface since there are some differences between both interface

Reply:
I don't have a strong opinion with this and i am fine with both options

Reply:
alright then i keep it as it is